### PR TITLE
feat: support 32k text-generation and multilingual embedding models

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: bigframes/ml/llm.py [hunk 1]
```diff
@@ -25,29 +25,38 @@
 import bigframes.pandas as bpd
 
 _REMOTE_TEXT_GENERATOR_MODEL_CODE = "CLOUD_AI_LARGE_LANGUAGE_MODEL_V1"
+_REMOTE_TEXT_GENERATOR_32K_MODEL_CODE = "text-bison-32k"
+_REMOTE_TEXT_GENERATOR_32K_MODEL_CODE = "text-bison-32k"
```
[the remark this answers is not in the document]
Comment (ashleyxuu) [reply bigframes/ml/llm.py:29]: My bad. Stuck with the merge conflicts. Will be more careful next time.

File: bigframes/ml/llm.py [hunk 2]
```diff
@@ -25,29 +25,38 @@
...
 _REMOTE_EMBEDDING_GENERATOR_MODEL_CODE = "CLOUD_AI_TEXT_EMBEDDING_MODEL_V1"
+_REMOTE_EMBEDDING_GENERATOR_MUlTILINGUAL_MODEL_CODE = "textembedding-gecko-multilingual"
 _EMBED_TEXT_RESULT_COLUMN = "text_embedding"
 
 
 class PaLM2TextGenerator(base.Predictor):
     """PaLM2 text generator LLM model.
 
     Args:
+        model_name (str, Default to "text-bison"):
```
[the remark this answers is not in the document]
Comment (ashleyxuu) [reply bigframes/ml/llm.py:41]: My bad. Changed the init to `text-bison`

File: bigframes/ml/llm.py [hunk 3]
```diff
@@ -118,7 +130,8 @@ def predict(
 
             top_k (int, default 40):
                 Top-k changes how the model selects tokens for output. A top-k of 1 means the selected token is the most probable among all tokens
-                in the model’s vocabulary (also called greedy decoding), while a top-k of 3 means that the next token is selected from among the 3 most probable tokens (using temperature).
+                in the model's vocabulary (also called greedy decoding), while a top-k of 3 means that the next token is selected from among the 3 most probable tokens (using temperature).
+                in the model's vocabulary (also called greedy decoding), while a top-k of 3 means that the next token is selected from among the 3 most probable tokens (using temperature).
```
[the remark this answers is not in the document]
Comment (ashleyxuu) [reply bigframes/ml/llm.py:134]: Done.

File: tests/system/small/ml/test_llm.py [hunk 4]
```diff
@@ -48,6 +54,30 @@ def test_create_text_generator_model_default_session(bq_connection, llm_text_pan
...
+    import bigframes.pandas as bpd
+
+    bpd.close_session()
+    bpd.options.bigquery.bq_connection = bq_connection
+    bpd.options.bigquery.location = "us"
+
+    model = llm.PaLM2TextGenerator(model_name="text-bison-32k")
+    assert model is not None
+    assert model._bqml_model is not None
+    assert model.connection_name.casefold() == "bigframes-dev.us.bigframes-rf-conn"
```
[the remark this answers is not in the document]
Comment (ashleyxuu) [reply tests/system/small/ml/test_llm.py:70]: In this case, it is not necessary to add the `casefold`, but I guess Garrett added this to be consistent with BQML docs: https://cloud.google.com/bigquery/docs/generate-text#create_a_model (They are using Capitalized letters for their SQL samples)

**Review by shobsi — APPROVED** [review]

**Pull request merged**